src/util/Util.js

```diff
@@ -46,6 +46,26 @@ class Util {
   }
 
   /**
+   * Format a string
+   * @param {string} format one of `bold`, `italic`, `underlined`, `strikethrough`
+   * @param {string} str String to format
+   * @returns {string} formatted string
+   */
+  static applyFormat(format, str) {
+    const formatters = {
```

Comment:
This should be defined outside of the method, like as a static property, or just a constant outside of the class.